soul-plugin/soul-plugin-ratelimiter/src/main/java/org/dromara/soul/plugin/ratelimiter/executor/RedisRateLimiter.java

```diff
@@ -44,9 +44,9 @@
     
     public static final String RATE_LIMITER_ALGORITHM = "rateLimiterAlgorithm";
     
-    public static final String KEYS = "keys";
+    public static final String RATE_LIMITER_KEYS = "keys";
```

Comment:
I think you misunderstood @tydhot 's meaning. That should be something like that:
`public static final String RATE_LIMITER_KEYS = "_rate_limiter_keys";`